[2025-03-05T07:46:22.359+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T07:46:22.368+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Stocks.load manual__2025-03-05T07:46:19.751562+00:00 [queued]>
[2025-03-05T07:46:22.373+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Stocks.load manual__2025-03-05T07:46:19.751562+00:00 [queued]>
[2025-03-05T07:46:22.373+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-05T07:46:22.380+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): load> on 2025-03-05 07:46:19.751562+00:00
[2025-03-05T07:46:22.385+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1669) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T07:46:22.385+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Stocks', 'load', 'manual__2025-03-05T07:46:19.751562+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/data226_hwk5 (7).py', '--cfg-path', '/tmp/tmp7dhdyn9w']
[2025-03-05T07:46:22.386+0000] {standard_task_runner.py:72} INFO - Started process 1672 to run task
[2025-03-05T07:46:22.387+0000] {standard_task_runner.py:105} INFO - Job 48: Subtask load
[2025-03-05T07:46:22.412+0000] {task_command.py:467} INFO - Running <TaskInstance: Stocks.load manual__2025-03-05T07:46:19.751562+00:00 [running]> on host ae5c3e699e69
[2025-03-05T07:46:22.460+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='Stocks' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T07:46:19.751562+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T07:46:19.751562+00:00'
[2025-03-05T07:46:22.461+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T07:46:22.467+0000] {base.py:84} INFO - Retrieving connection 'snowflake_conn'
[2025-03-05T07:46:22.468+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-5.15.167.4-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2025-03-05T07:46:22.469+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-05T07:46:22.469+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-05T07:46:23.185+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T07:46:23.359+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T07:47:16.092+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T07:47:16.093+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T07:47:16.098+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T07:47:16.099+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Stocks, task_id=load, run_id=manual__2025-03-05T07:46:19.751562+00:00, execution_date=20250305T074619, start_date=20250305T074622, end_date=20250305T074716
[2025-03-05T07:47:16.125+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T07:47:16.133+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-05T07:47:16.135+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
